code/game/turfs/walls/wall_types.dm

```diff
@@ -368,6 +368,11 @@
 	if(!locate(/obj/effect/alien/weeds) in loc)
 		new /obj/effect/alien/weeds(loc)
 
+/turf/closed/wall/resin/ChangeTurf(new_turf_path, forget_old_turf, flags)
+	. = ..()
+	if(forget_old_turf)
+		new /obj/effect/alien/weeds(src)
```

Comment:
Due to how turfs work, the new turf will have deleted the current one and thus, But since the parent version has a return of said new turf:
```suggestion
		new /obj/effect/alien/weeds(.)
```
doing this should possibly work.

Reply:
Testing this it did work locally, but i agree its prob referencing the wrong turf.

Reply:
Mh, that makes me wonder even more about how damned undocumented some standard features of byond are.